clustered/ehcache-client/src/main/java/org/ehcache/clustered/client/internal/store/ClusteredStore.java

```diff
@@ -554,6 +554,11 @@ public Map<K, ValueHolder<V>> bulkComputeIfAbsent(final Set<? extends K> keys, f
     }
   }
 
+  @Override
+  public Iterable<? extends Map.Entry<? extends K, ? extends ValueHolder<V>>> bulkComputeIfAbsentAndFault(Iterable<? extends K> keys, Function<Iterable<? extends K>, Iterable<? extends Map.Entry<? extends K, ? extends V>>> mappingFunction) throws StoreAccessException {
+    return bulkComputeIfAbsent((Set<? extends K>) keys,mappingFunction).entrySet();
```

Comment:
This doesn't fix the issue, it just masks it. When the broken runs the `bulkComputeIfAbsent(...)` method throws the `UnsupportedOperationException` at line 553 which happens to allow the test to pass, but it's not passing for the right reasons.

Reply:
This was the broken test - CacheLoaderWithoutReadThroughTest#shouldPropagateExceptionUsingLoadAll()

I have observed below behavior of broken test by running it in master branch(without my changes)
This test has the exact same behavior (which you have mentioned here), where it is getting failed at line 553 and throws `UnsupportedOperationException ` then validating the exception cause in test by -  `assertThat(e.getCause(), instanceOf(CacheLoaderException.class));`

After applying my fix for this test, the test is still behaving same and the same exception hierarchy is been thrown 
```
----issue-3097 branch----------
java.util.concurrent.ExecutionException:
javax.cache.integration.CacheLoaderException:
java.lang.UnsupportedOperationException:
This bulkComputeIfAbsent method is not yet capable of handling generic computation functions

----master branch-------
java.util.concurrent.ExecutionException:
javax.cache.integration.CacheLoaderException:
java.lang.UnsupportedOperationException:
This bulkComputeIfAbsent method is not yet capable of handling generic computation functions
```